src/OSPSuite.Presentation/Presenters/Importer/ColumnMappingPresenter.cs

```diff
@@ -134,10 +139,26 @@ public void SetRawData(UnformattedData rawData)
 
       public void SetDescriptionForRow(ColumnMappingDTO model)
       {
-         var values = _metaDataCategories.FirstOrDefault(md => md.Name == model.MappingName)?.ListOfValues.Select(v => v.ToString());
+         var values = _metaDataCategories.FirstOrDefault(md => md.Name == model.MappingName)?.ListOfValues.Keys.Select(v => v);
          _setDescriptionForRow(model, values != null && values.All(v => v != model.ExcelColumn));
       }
 
+      public void UpdateMetaDataForModel()
+      {
+         if (_currentModel == null)
+            return;
+         var metaData = _currentModel.Source as MetaDataFormatParameter;
+         if (metaData == null)
+         {
```

Comment:
remove curly braces since 2 lines before you are not using it

Reply:
done